src/server/data/publications-data.json

```diff
@@ -0,0 +1 @@
+[{"name":"321","description":"321","year":"651","authors":"321","topic":"321","type":"321","publisher":"321","publication status":"54","fileLinks":[{"info":"66","type":"44","link":"2"},{"info":"3","type":"1","link":"2"}]},{"name":"l","description":"k","year":"k","authors":"k","topic":"j","type":"kn","publisher":"kn","publication status":"j","fileLinks":[{"info":"8","type":"jh","link":"uh"}]}]
```

Comment:
I am not sure this is the right format

Reply:
it seems to match the academic-publications.json file, except for the publication-status field that I need to fix. am I missing something?